helm-chart/kuberay-apiserver/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
```

Comment:
@kevin85421 I'm not sure about this one.  Can you confirm if this is supposed to be updated?

Reply:
Yes